c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -325,7 +291,7 @@ enum callback_priorities {
 // ------------------------------------------------------------------------
 
 // The content of the communications controller SAR register
```

Comment:
that your going to set. not the actual contents

Reply:
I think I'm using the right names from the datasheet now.